signer/src/signature.rs

```diff
@@ -0,0 +1,33 @@
+//! Mina signature structure and associated helpers
+
+use std::fmt;
+
+use crate::{BaseField, FieldHelpers, ScalarField};
+
+/// Signature structure
+#[derive(Clone, Copy, Eq, fmt::Debug, PartialEq)]
+pub struct Signature {
+    /// Field component
+    pub rx: BaseField,
+
+    /// Scalar component
+    pub s: ScalarField,
```

Comment:
do these fields need to be public outside of the crate? If not you can set `pub(crate)`

Reply:
Some things in another crate (the tests and the websnapps prover) need to access them individually.   Seems like something that should be public.